test/config-luatest/rpc_test.lua

```diff
@@ -15,6 +16,24 @@ local function skip_if_no_vshard()
     t.skip_if(not has_vshard, 'Module "vshard-ee/vshard" is not available')
 end
 
+local function start_stub_servers(g, dir, instances)
+    g.stub_servers = g.stub_servers or {}
+    for _, instance in ipairs(instances) do
+        local uri = ('%s/%s.iproto'):format(dir, instance)
+        local s = socket.tcp_server('unix/', uri, function(s)
+            s:detach()
```

Comment:
This way `:close()` in `stop_stub_servers` will not close the file descriptor, I guess. I think that we can just sleep the fiber here.